wxcamwindow/camwindow.cpp

```diff
@@ -158,29 +158,21 @@ void CCamWindow::DrawCam (IplImage* pImg)
 		m_ImageCopyMutex.Leave();
 		
 		// Check that image is RGB with channel order RGB or BGR
-		wxASSERT_MSG 
-			(pImg->nChannels== 3 &&
-			((pImg->channelSeq[0]== 'R' && pImg->channelSeq[1]== 'G' && pImg->channelSeq[2]== 'B') ||
-			(pImg->channelSeq[0]== 'B' && pImg->channelSeq[1]== 'G' && pImg->channelSeq[2]== 'R')),
+		wxASSERT_MSG (
```

Comment:
We don't have any information about the format here. Does the assertion make any sense?

Reply:
I don't think so.